source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service_manager.py

```diff
@@ -182,18 +195,18 @@ def decorate_journeys_with_ridesharing_offers(self, response, request, instance)
                             period_extremity = PeriodExtremity(section.end_date_time, False)
                         else:  # ridesharing at the end, we search for solution starting after the end of the pt sections
                             period_extremity = PeriodExtremity(section.begin_date_time, True)
-
+                        instance_params = self.InstanceParams.make_params(instance)
```

Comment:
you can probably call it here : https://github.com/CanalTP/navitia/pull/3335/files#diff-58a666992e45cd17e1e1da8366f23cff7f1d8cbb03b84102293225f827713b04R174
At the top of the function ? Like that `greenlet_pool_actived` will be fill by self.InstanceParams.make_params(instance) Class